src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPage.java

```diff
@@ -0,0 +1,414 @@
+      return false;
+    }
+  }
+
+  public boolean isMessageOnChat() {
+    try {
+      return userPostedMessage().isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isUserWelcomeMessageDisplayed(String userName) {
+    try {
+      return userWelcomeMessage(userName).isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isPrivateMessageHeaderDispayed() {
+    try {
+      return privateMessagesHeader.isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isPermissionsErrorTitleDisplayed() {
+    try {
+      return permissionsErrorTitle.isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isPrivateMessageNotificationDisplayed() {
+    try {
+      return privateMessageNotification.isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isPricvateNotificationCountDisplayed(int notificationCount) {
+    try {
+      return userPrivateMessageNotification(notificationCount).isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isPrivateChatOpen() {
+    try {
+      return privateChatHeader.isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isRegularUserDropdownDisplayed() {
+    if (userDropDownActionsElements.size() == REGULAR_USER_DROPDOWN_ELEMENTS) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  public boolean isUserInPrivateSectionDisplayed(String userName) {
+    try {
+      return userOnChat(userName, PRIVATE_MESSAGE_USER_SELECTOR).isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isChatUnbanMessageDispalyed() {
+    try {
+      return getUserUnbanMessage().isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isMessageWallOpened(String userName) {
+    try {
+      Assertion.assertEquals(userName, USER_NAME_ON_PROFILE_SCREEN);
+      return userPageMessageWallTab.isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isContributionsPageOpened(String userName) {
+    try {
+      Assertion.assertEquals(userName, USER_NAME_ON_PROFILE_SCREEN);
+      return userPageContributionsTab.isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isUserKickedFromChat() {
+    try {
+      return userBlockedMessageField.isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isBlockPrivateMessageButtonDisplayed() {
+    try {
+      return blockPrivateMassageButton.isDisplayed();
+    } catch (ElementNotFoundException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)